src/main/java/scripting/AbstractPlayerInteraction.java

```diff
@@ -133,6 +133,16 @@ public void warpParty(int id, int portalId) {
         warpParty(id, portalId, mapid, mapid);
     }
 
+    public void warpParty(int map, String portalName) {
+
+        int mapid = getMapId();
+        var warpMap = c.getChannelServer().getMapFactory().getMap(map);
+        var portal = warpMap.getPortal(portalName).getId();
```

Comment:
Missing null check on the obtained portal. There may not be a portal with the provided name, and then `.getId()` will trigger a NullPointerException.

Reply:
Added the null check and had it mirror the behavior of the changeMap for an individual character. It will get the default portal 0 as the backup if the string snags a null portal. This is the existing behavior for all map changing on the Character.java